src/main/kotlin/org/cultofclang/bonehurtingjuice/Extensions.kt

```diff
@@ -16,6 +18,11 @@ internal val Block.isFlowing: Boolean
         val blockData = blockData
         return (blockData is Levelled) && blockData.level >= 8
     }
+internal val Block.isBubbleColumn: Boolean
+    get() {
+        val blockData = blockData
```

Comment:
why the separate statements here?

```kotlin
internal val Block.isBubbleColumn: Boolean = blockData is BubbleColumn
```
should be enough

Reply:
`internal val Block.isBubbleColumn: Boolean = blockData is BubbleColumn` didnt work but i did shorten it :)